lib/core/src/sdk.rs

```diff
@@ -1031,6 +1046,17 @@ impl LiquidSdk {
             compressed: true,
             inner: refund_keypair.public_key(),
         };
+        let webhook = self.persister.get_webhook_url()?.map(|url| ChainWebhook {
+            url,
+            hash_swap_id: Some(true),
+            status: Some(vec![
+                ChainSwapStates::SwapExpired,
```

Comment:
Do we get SwapExpired only if we broadcasted the lockup or even for swaps that we didn't start yet?
We need to make sure the user is not spammed.

Reply:
We also get if the lockup fails. I think it's enough to handle TransactionLockupFailed and TransactionLockupFailed